### DISCUSSION TITLE
Fields sanitization and output escaping

### DISCUSSION THREAD

**Discussion by NazarTikhoniuk** [wpengine/atlas-content-modeler#608] [Q&A / Help / Support] [answered]
Greetings!
Can you, please, advise if fields that are created in the Atlas Content Modeler are properly sanitized and escaped?
Thanks!

**Comment by nickcernis** [comment, marked as answer]
Thanks for the question, @NazarTikhoniuk!

Yes, ACM sanitizes input when saving field values. You can see the sanitization code here if you'd like to:

https://github.com/wpengine/atlas-content-modeler/blob/b9f06ee9fb3c389c9817002fcd0f25664a92e956/includes/publisher/lib/field-functions.php#L261
